pkg/kube/pod.go

```diff
@@ -349,3 +354,17 @@ func strategicMergeJsonPatch(original, override interface{}) ([]byte, error) {
 	}
 	return mergedPatch, nil
 }
+
+// getPodReadyWaitTimeout returns the pod ready wait timeout from ENV if configured
+// returns the default of 15 minutes otherwise
+func getPodReadyWaitTimeout() time.Duration {
+	if v, ok := os.LookupEnv(podReadyWaitTimeoutEnv); ok {
+		iv, err := strconv.Atoi(v)
+		if err == nil {
+			return time.Duration(iv) * time.Minute
+		}
+		log.Debug().Print("Using default timeout value because of invalid environment variable", field.M{"envVar": v})
+	}
+
+	return defaultPodReadyWaitTimeout
+}
```

Comment:
```suggestion
// GetPodReadyWaitTimeout returns the pod ready wait timeout from ENV if configured
// returns the default of 15 minutes otherwise
func GetPodReadyWaitTimeout() time.Duration {
	if v, ok := os.LookupEnv(PodReadyWaitTimeoutEnv); ok {
		iv, err := strconv.Atoi(v)
		if err == nil {
			return time.Duration(iv) * time.Minute
		}
		log.Debug().Print("Using default timeout value because of invalid environment variable", field.M{"envVar": v})
	}

	return DefaultPodReadyWaitTimeout
}
```